### PR TITLE
docs(auto-generating-selectors): Add vanilla store variant

### PR DESCRIPTION
**Pull request by luizeboli**
## Summary

The proposed change came from this [discussion](https://github.com/pmndrs/zustand/discussions/1637#discussioncomment-4989111) . The idea is to centralize both possibilities in the same wiki page.

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

File: docs/guides/auto-generating-selectors.md [hunk 1]
```diff
@@ -65,6 +65,54 @@ const bears = useBearStore.use.bears()
 const increment = useBearStore.use.increment()
 ```
 
+## Vanilla Store
+
+If you are using a vanilla store, then consider the function below.
```
Comment (sewera) [inline docs/guides/auto-generating-selectors.md:70]:
```suggestion
If you are using a vanilla store, use the following approach:
```

is more in line with the general tone and style of this page.

File: docs/guides/auto-generating-selectors.md [hunk 2]
```diff
@@ -65,6 +65,54 @@ const bears = useBearStore.use.bears()
...
+const createSelectors = <S extends StoreApi<object>>(_store: S) => {
+  const store = _store as WithSelectors<typeof _store>;
+  store.use = {};
+  for (const k of Object.keys(store.getState())) {
+    (store.use as any)[k] = () =>
+      useStore(_store, (s) => s[k as keyof typeof s]);
+  }
+
+  return store;
+};
```
Comment (sewera) [inline docs/guides/auto-generating-selectors.md:87]: The code style is inconsistent with Zustand's one. Please adjust according to the code snippets above.

File: docs/guides/auto-generating-selectors.md [hunk 3]
```diff
@@ -65,6 +65,54 @@ const bears = useBearStore.use.bears()
...
+  increment: () => set((state) => ({ bears: state.bears + 1 })),
+}))
+
+const bearStoreWithHooks = createSelectors(bearStoreBase);
+
+// get the property
+const bears = bearStoreWithHooks.use.bears()
+
+// get the action
+const increment = bearStoreWithHooks.use.increment()
```
Comment (sewera) [inline docs/guides/auto-generating-selectors.md:113]: Also code style in this snippet is inconsistent. Please adjust.

**Review by sewera — CHANGES_REQUESTED** [review]

File: docs/guides/auto-generating-selectors.md (hunk 2 above)
Comment (luizeboli) [reply docs/guides/auto-generating-selectors.md]: Hi @sewera! I did some refactoring trying to match Zustand code style based on the above snippets. Can you check again?

**Review by sewera — CHANGES_REQUESTED** [review]
You didn't run `yarn prettier`.

ss_20230704_225747

Please don't check the checkbox in the PR that you've run it if you haven't. Please run `yarn prettier` the next time.

If you are unsure, please run it every time before you commit. It will save the time of the reviewers and yours as well. After all, it's our free time, so let's delegate the boring stuff to the machines.

**Comment by luizeboli** [comment]
@sewera You are right, I'm sorry. I forgot to run it on my last commit.

Now I think it's ok.

**Review by sewera — APPROVED** [review]

**Pull request merged**